compiler/luci/pass/src/FuseBCQPass.cpp

```diff
@@ -268,7 +289,7 @@ bool FuseBCQPass::run(loco::Graph *g)
         bcq_gather->input_clusters(converter.packed_clusters(params));
 
         const auto binary_hidden_size =
-            loco::must_cast<luci::CircleConst *>(bcq_gather->input_binary())->dim(0).value() * 32;
+            loco::must_cast<luci::CircleConst *>(bcq_gather->input_binary())->dim(1).value() * 32;
```

Comment:
what is 0 and 1? please add a comment what this index is.

Reply:
I will add :)
But should I apply this to 1.7.0 too?

Reply:
> But should I apply this to 1.7.0 too?

Yes please